3-platform/shopping-cart/app/src/main/java/com/_6core/platform/shopping/cart/app/model/ShoppingCart.java

```diff
@@ -0,0 +1,16 @@
+package com._6core.platform.shopping.cart.app.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
Did you test that your mapping work? 
As I remember NoArgsConstructor is required for entity. 
```suggestion
@NoArgsConstructor
```

Reply:
Ask @VictoriaKh-T about how you can verify mapping.

Reply:
Yes, "@NoArgsConstructor", thx